modules/core/src/main/scala/lucuma/odb/api/schema/ConstraintSetSchema.scala

```diff
@@ -25,7 +24,7 @@ object ConstraintSetSchema {
   implicit val EnumTypeWaterVapor: EnumType[WaterVapor] =
     EnumType.fromEnumerated("WaterVapor", "Water vapor")
 
-  def AirMassRangeType[F[_]]: ObjectType[OdbRepo[F], AirmassRange] =
+  val AirMassRangeType: ObjectType[Any, AirmassRange] =
```

Comment:
Why not just use `F[_]`?

Reply:
The context object is not used so it seemed better to allow any context than to require an `OdbCtx[F]` or `OdbRepo[F]`.